src/backup-all-indexes.sh

```diff
@@ -100,6 +109,9 @@ for index_name in $(curl -sS ${DATABASE_URL}/_cat/indices | grep logstash- | sed
       else
         echo "$(now): ${index_name} archival failed."
       fi
+      if [[ "${index_name:9}" > "${DELETE_CUTOFF_DATA}" ]]; then
```

Comment:
Should the deletion be performed in all cases, even if the archival failed?

Reply:
fine point!